packages/suite/src/views/wallet/tokens/hidden-tokens/components/HiddenTokensTable.tsx

```diff
@@ -67,10 +82,15 @@ export const HiddenTokensTable = ({ selectedAccount, searchQuery }: HiddenTokens
             )}
             {unverifiedTokensCount > 0 && (
                 <UnverifiedTokens>
-                    <QuestionTooltip
-                        label="TR_TOKEN_UNRECOGNIZED_BY_TREZOR"
-                        tooltip="TR_TOKEN_UNRECOGNIZED_BY_TREZOR_TOOLTIP"
-                    />
+                    <UnrecognizedTokensWrapper>
+                        <H3>
+                            <Translation id="TR_TOKEN_UNRECOGNIZED_BY_TREZOR" />
+                        </H3>
+                        <SubHeading>
```

Comment:
Could you refactor it this way?

```tsx

import { Text, Row } from '@trezor/components';

...

<Row alignItems="center" gap={spacingsPx.xxs} margin={{bottom: spacingsPx.xl, top: spacingsPx.xxs}}>
  <Icon variant="tertiary" size={16} icon="WARNING" />
  <Text variant="tertiary" typographyStyle="hint">
    <Translation id="TR_TOKEN_UNRECOGNIZED_BY_TREZOR_TOOLTIP" />
  </Text>
</Row>
```

Reply:
260e50a74be6560cb7f820b01e1e4c9c98fa84b1